src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.html

```diff
@@ -26,8 +26,8 @@
               {{ dashboard.name }}
             </td>
             <td title="{{ dashboard.full_name }}">{{ $ctrl.currency(dashboard) }}</td>
-            <td title="{{ dashboard.full_name }}">{{ dashboard.updated_at | date: "dd/MM/yyyy \'at\' h:mm a" }}</td>
-            <td title="{{ dashboard.full_name }}">{{ dashboard.created_at |  date: "dd/MM/yyyy \'at\' h:mm a" }}</td>
+            <td title="{{ dashboard.full_name }}">{{ dashboard.updated_at | amDateFormat:"L [at] h:mm a" }}</td>
```

Comment:
We should get rid of the `[at]` as it's not translated, just escaped.
"L LT" should be fine?

Reply:
👍 yeah agreed the `[at]` not great, will check if` L LT` works.